src/controller/genericmidi.cxx

```diff
@@ -623,28 +398,30 @@ void GenericMIDI::launchScene( int scene )
     
     if ( b->action == GRID_LAUNCH_SCENE )
     {
-      for( int i = 0; i < 5; i++ )
```

Comment:
Can we remove this loop without any nasty side-effects? I think its needed to reset the state of each grid-square on the MIDI controller, before updating the newest state...? Not sure, but it looks suspect.